nnvm/python/nnvm/frontend/__init__.py

```diff
@@ -7,3 +7,7 @@
 from .darknet import from_darknet
 from .tensorflow import from_tensorflow
 from .caffe2 import from_caffe2
+from .common import raise_not_supported, get_nnvm_op, required_attr, \
```

Comment:
is this redundant?

Reply:
`raise_not_supported` is certainly redundant and has been removed. `get_nnvm_op` and `required_attr` are helper functions that invoke error handling but ultimately return various objects such as symbols or attributes.